config.def.h

```diff
@@ -376,6 +376,10 @@ static MouseShortcut mshortcuts[] = {
 	{ XK_ANY_MOD,           Button4, ttysend,        {.s = "\031"} },
 	{ XK_ANY_MOD,           Button5, ttysend,        {.s = "\005"} },
 	#endif // SCROLLBACK_MOUSE_ALTSCREEN_PATCH
+	#if OSC133_PATCH
+	{ TERMMOD,              XK_Z,           scrolltoprompt,  {.i = -1}, S_PRI },
+	{ TERMMOD,              XK_X,           scrolltoprompt,  {.i =  1}, S_PRI },
+	#endif // OSC133_PATCH
```

Comment:
This is in the wrong place. It should be in the keyboard shortcuts.

Reply:
Thanks. I've fixed it.